client/lxd_oidc.go

```diff
@@ -115,11 +115,22 @@ func (o *oidcClient) do(req *http.Request) (*http.Response, error) {
 	issuer := resp.Header.Get("X-LXD-OIDC-issuer")
 	clientID := resp.Header.Get("X-LXD-OIDC-clientid")
 	audience := resp.Header.Get("X-LXD-OIDC-audience")
```

Comment:
@tomponline I've never liked that we use headers for this. The API documentation for the [`/1.0?public` endpoint](https://documentation.ubuntu.com/lxd/en/latest/api/#/server/server_get_untrusted) says:

> Shows a small subset of the server environment and configuration which is required by untrusted clients to reach a server.

IMO we should have a small subset of config available over /1.0 for untrusted clients, including these values.